### ISSUE TITLE
Native HDHomeRun Apps

### ISSUE THREAD

**Issue by smartinec** [craigmox/cetonproxy#26]
I tried downloading the HDHomeRun Windows 10 App (https://www.microsoft.com/en-us/p/hdhomerun/9nblggh58vwk) to the same machine as Ceton Proxy, but the app says it can't find an HDHomeRun tuner. I think it's setup correctly because I have Plex working fine. Any ideas?

**Comment by smartinec** [comment]
1 little update: I put static IPs into the Ceton Proxy advanced settings and now the HDHomeRun Setup tool is giving "Discovery Error 4002".

**Comment by rbknox** [comment]
It seems “auto” tuner recognition by most dvr apps doesn’t work.  I am able to play a stream to vlc.  I can also make things work in Emby by specifying the ip:5004.   HDHomeRun setup either doesn’t see the Proxy tuners or gives the discovery error (when IPs are added to proxy advanced settings.  I don’t use Plex; I use WMC - will try that next though others have reported no detection.

**Comment by mpatriche** [comment]
Hi.  CetonProxy may not emulate enough functionality to work with the HDHomeRun app.  The logs from CetonProxy and the HDHomeRun app may offer some detail as to what is happening.  Just curious what functionality you're looking for from the HDHomeRun app?  I used Emby with WMC as my LiveTV source.  But with CetonProxy, I have completely removed my dependency on WMC, and Emby handles all of my DVR functionality.

**Comment by rbknox** [comment]
Yeah, just looking for options as I consider the end of WMC -- still my main dvr, running on Windows 8.1;  I have Emby running but don't like the guide much.  Also trying the HDHR app (I have  Prime and a new 4K OTA ).   The CetonProxy does allow me to salvage the Ceton Infinitv 4, but I was also looking for a way to centralize it, perhaps on a Win10 server, sort of as a half step towards shutting down WMC.  The idea that the Ceton would appear as a HDHR tuner was appealing, but I do understand it was really a Plex workaround.   Still, seems it wouldn't take much to emulate enough that Ceton becomes a discoverable device.
